libsolidity/analysis/TypeChecker.cpp

```diff
@@ -3212,6 +3222,20 @@ bool TypeChecker::visit(MemberAccess const& _memberAccess)
 				if (contractType && contractType->isSuper())
 					requiredLookup = VirtualLookup::Super;
 			}
+
+		if (
+			funType->kind() == FunctionType::Kind::Send ||
+			funType->kind() == FunctionType::Kind::Transfer
+		)
+			m_errorReporter.warning(
+				9207_error,
+				_memberAccess.location(),
+				fmt::format(
+					"'{}' is deprecated and scheduled for removal in the next breaking version (0.9). "
+					"You can use 'call{{value: <amount>}}()' instead.",
```

Comment:
optional, i just think this reads a bit nicer:

```suggestion
					"It is encouraged to use 'call{{value: <amount>}}()' instead.",
```

Reply:
Done.